src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import static core.basesyntax.ColorSupplier.getRandomColor;
```

Comment:
Avoid using static methods as per the checklist. Instead of calling a static method from ColorSupplier, you should create an instance of ColorSupplier and use its methods.